### PR TITLE
Changed HSB to HSL color model

### PR DESCRIPTION
**Pull request by rpetrusha**
# Changed HSB to HSL color model

@Tanya-Solyanik

### CODE REVIEW

File: xml/System.Drawing/Color.xml [hunk 1]
```diff
@@ -2163,7 +2163,7 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>Gets the hue-saturation-brightness (HSB) brightness value for this <see cref="T:System.Drawing.Color" /> structure.</summary>
+        <summary>Gets the hue-saturation-lightness (HSL) brightness value for this <see cref="T:System.Drawing.Color" /> structure.</summary>
```
Comment (Tanya-Solyanik) [inline xml/System.Drawing/Color.xml]: As far as I understand, Brightness and Lightness are not used interchangeably, so HSL color model does not have "brightness".  I know it sounds ridiculous "GetBrightness" returns lightness, but I think it is more accurate.

**Review by mairaw — APPROVED** [review]
LGTM once you address Tanya's comment.

**Review by Tanya-Solyanik — CHANGES_REQUESTED** [review]
I think docs should say that GetBrightness returns a lightness value....

File: xml/System.Drawing/Color.xml (hunk 1 above)
Comment (rpetrusha) [reply xml/System.Drawing/Color.xml]: I should have known, @Tanya-Solyanik. I couldn't find any information on brightness in the HSL model.

**Comment by rpetrusha** [comment]
I've addressed your comment, @Tanya-Solyanik, if you want to take another look.

**Review by Tanya-Solyanik — APPROVED** [review]

**Pull request merged**